src/Microsoft.PowerApps.TestEngine/PowerFx/PowerFxHelper.cs

```diff
@@ -40,10 +42,13 @@ public static IEnumerable<string> ExtractFormulasSeparatedByChainingOperator(Eng
         /// </summary>
         /// <param name="engine">Instance of an engine configured with the desired locale</param>
         /// <param name="expression">Expression from which formulas separated by chaining operator would be extracted</param>
+        /// <param name="culture">The locale to be used when excecuting tests</param>
         /// <returns>Spans that represent formulas separated by chaining operator at multiple levels and depths</returns>
-        private static IEnumerable<Span> ExtractSpansOfFormulasSeparatedByChainingOperator(Engine engine, string expression)
+        private static IEnumerable<Span> ExtractSpansOfFormulasSeparatedByChainingOperator(Engine engine, string expression, CultureInfo culture = null)
         {
-            var chainOperatorTokens = engine.Tokenize(expression).Where(tok => tok.Kind == TokKind.Semicolon).OrderBy(tok => tok.Span.Min);
+
+            var resultp = engine.Tokenize(expression, culture);
```

Comment:
Why called tokenize twice? looks like resultp is unused

Reply:
Ouch that was while debugging it was not used anywhere. Let me remove that.